.gitignore

```diff
@@ -2,3 +2,5 @@
 *.phar
 composer.lock
 vendor
+.idea
```

Comment:
this one must be removed. It is specific to your system, not to the project, so it should not be added in the project-level ignore rules. See https://help.github.com/articles/ignoring-files/ to know how to ignore it on your system